docs/guides/server/caching.adoc

```diff
@@ -349,6 +349,25 @@ To set up for IPv6 only and have {project_name} pick the bind address automatica
 export JAVA_OPTS_APPEND="-Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true"
 ----
 
+== Running behind a firewall or reverse proxy
+
+If you are running behind a firewall or with port forwarding, the IP address and port that other nodes of a {project_name} cluster should use to connect to an instance might be different from the IP address and port your {project_name} instance binds to.
+In such a case {project_name} needs additional hints to advertise the IP address and port other nodes should use to connect to this instance.
+For this scenario, use the following properties:
+
+|===
+|Property | Description
+
+m|jgroups.external_port
+|Port that other instances in the {project_name} cluster should use to contact this node.
+
+m|jgroups.external_addr
+|IP address that other instances in the {project_name} should use to contact this node.
+
+|===
+
+NOTE: Use `-D<property>=<value>` set this in your `JAVA_OPTS_APPEND` environment variable or in your CLI command.
```

Comment:
In the previous section we tell users to configure this as:

```
[source,bash]
----
export JAVA_OPTS_APPEND="-Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true"
----
```

I think we should be consistent and use the same format here.

Reply:
I'd say the one you picked is special as it shows how you used those properties that we didn't introduce earlier. 

The documentation snippet I used is consistent with https://github.com/keycloak/keycloak/blob/1932fbcbb11fec50ecdfcfd6a753371b1edc78f2/docs/guides/server/caching.adoc?plain=1#L332

May I suggest that it is good enough until we have full CLI options for these items?

Reply:
Sure, I didn't look at the whole doc so I didn't realise we had already used this convention :+1: